src/main/scala/com/fulcrumgenomics/bam/ClipOverlappingReads.scala

```diff
@@ -117,6 +121,12 @@ class ClipOverlappingReads
     else {
       val (f,r) = if (r1.getReadNegativeStrandFlag) (r2, r1) else (r1, r2)
 
+      // CLip the 5' end of both reads if desired
+      if (this.numStartClip > 0) {
```

Comment:
Note that adding this here means it will not clip any reads that are not F/R pairs on the same chromosome.  That seems ok if this is only used to feed the consensus pipeline, but it's a general tool so should probably clip all reads.